src/Rules/RuleRunner.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Core.Bases;
+using Axe.Windows.Core.Enums;
+using Axe.Windows.Telemetry;
+using System;
+using System.Collections.Generic;
+using static System.FormattableString;
+
+namespace Axe.Windows.Rules
+{
+    class RuleRunner
```

Comment:
There's already RuleSelection.RuleRunner class, and adding Rules.RuleRunner could lead to some potential confusion. Should we consider a different name?

Reply:
I think the benefit of having an accurately named class outweighs the potential for confusion. I also think namespaces go a long way toward clarifying which class is being used.